scripts/cert/README.md

```diff
@@ -0,0 +1,10 @@
+# cert
+
+Scanner and ScannerDB requires valid certificates to run.
+`gen-cert.sh` generates certificates for these deployments
+and overwrites the current values in a given Kubernetes secret configuration file.
+
+To run from the top-level directory and overwrite `chart/templates/mock-scanner[-db]-tls.yaml`:
+```sh
+./scripts/cert/gen-cert.sh chart/templates/mock-scanner-tls.yaml chart/templates/mock-scanner-db-tls.yaml
+```
```

Comment:
Is it worth mentioning `cfssl` is a pre-req and how to install it? 

Something like: 

> Requires [`cfssl`](https://github.com/cloudflare/cfssl) to run, install by running:  `go install github.com/cloudflare/cfssl/cmd/...@latest`